### PR TITLE
feat: add meta tags to home page

### PR DESCRIPTION
**Pull request by WojtekTheWebDev**
add meta tags to home page

## Description

add meta tags with descriptions to home page, use titleTemplate in `nuxt.config.js`

## Related Issue

BC-303

## Motivation and Context

Meta tags for DEMO

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: packages/theme/pages/Home.vue [hunk 1]
```diff
@@ -225,6 +226,46 @@ export default defineComponent({
       toggleNewsletterModal();
     };
 
+    const metaTags = computed(() => ({
```
Comment (Baroshem) [inline packages/theme/pages/Home.vue:229]: Could you explain why you have set the meta tags as a computed property? From what I see it is rather static so I would register it statically

File: packages/theme/pages/Home.vue [hunk 2]
```diff
@@ -233,7 +274,8 @@ export default defineComponent({
       products,
       loading
     };
-  }
+  },
+  head: {}
```
Comment (Baroshem) [inline packages/theme/pages/Home.vue:278]: Is it necessary?

**Review by Baroshem — COMMENTED** [review]
Two comments

File: packages/theme/pages/Home.vue (hunk 2 above)
Comment (WojtekTheWebDev) [reply packages/theme/pages/Home.vue:278]: Unfortunatelly it is: https://composition-api.nuxtjs.org/packages/usemeta/

**Review by Baroshem — APPROVED** [review]

**Pull request merged**